lightly_solution.ipynb

```diff
@@ -11,318 +11,108 @@
-      "\n",
-      "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m A new release of pip is available: \u001b[0m\u001b[31;49m23.2.1\u001b[0m\u001b[39;49m -> \u001b[0m\u001b[32;49m23.3.2\u001b[0m\n",
-      "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m To update, run: \u001b[0m\u001b[32;49mpip install --upgrade pip\u001b[0m\n"
+      "Cloning into './dataset_clothing_images'...\n"
      ]
-    }
-   ],
-   "source": [
-    "\"\"\" Install the Lightly Python SDK. \"\"\"\n",
-    "!pip3 install lightly"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 6,
-   "metadata": {},
-   "outputs": [
+    },
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "LIGHTLY_WORKER_ID = 65806b455ca68c93b29ad6b3\n"
+      "Resolving deltas: 100% (7/7), done.\n",
+      "\u001b[01;34m./dataset_clothing_images\u001b[00m\n",
+      "├── LICENSE\n",
+      "├── README.md\n",
+      "├── \u001b[01;34mdress\u001b[00m [32 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34mhat\u001b[00m [14 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34mlongsleeve\u001b[00m [49 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34moutwear\u001b[00m [24 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34mpants\u001b[00m [49 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34mshirt\u001b[00m [29 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34mshoes\u001b[00m [26 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34mshorts\u001b[00m [25 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34mskirt\u001b[00m\n",
+      "│   ├── \u001b[01;35m2459a9ab-b654-4aef-be7c-89340da78cec.jpg\u001b[00m\n",
+      "│   ├── ...\n",
+      "│   └── \u001b[01;35mecdc68a9-12c7-449b-ab3a-4b2a8e71b197.jpg\u001b[00m\n",
+      "└── \u001b[01;34mt-shirt\u001b[00m [81 entries exceeds filelimit, not opening dir]\n",
+      "\n",
+      "10 directories, 14 files\n"
      ]
     }
    ],
    "source": [
-    "\"\"\" Register the Lightly Worker. \"\"\"\n",
-    "\n",
-    "from lightly.api import ApiWorkflowClient\n",
-    "\n",
-    "client = ApiWorkflowClient(token=lightly_token)\n",
-    "\n",
-    "# Create a Lightly Worker. If a worker with this name already exists, the id of the existing\n",
-    "# worker is returned.\n",
-    "worker_id = client.register_compute_worker(name=\"clothing-worker\")\n",
-    "\n",
-    "os.environ[\"LIGHTLY_WORKER_ID\"] = worker_id\n",
-    "!echo \"LIGHTLY_WORKER_ID =\" $LIGHTLY_WORKER_ID"
+    "# Set the path to the dataset.\n",
+    "# Here we downlaod and use a dataset containing 354 images showing clothes.\n",
+    "# If you want to use your own dataset, just set the dataset_path to it.\n",
+    "dataset_path = \"./dataset_clothing_images\"\n",
```

Comment:
Is there a particular reason you use `./dataset_clothing_images` instead of just `dataset_clothing_images`? I am asking because the former will only work on linux and mac but not on windows while the latter should work on all systems.

Reply:
Yes, because otherwise the worker does mount it correctly.

Reply:
But now you could use `dataset_clothing_images` because it is anyways converted to an absolute path for docker no?

Reply:
Yes, sorry, solved it